boards/nxp/frdm_mcxn947/board.c

```diff
@@ -7,6 +7,9 @@
 #include <fsl_clock.h>
 #include <fsl_spc.h>
 #include <soc.h>
+#if defined(CONFIG_ADC_MCUX_LPADC)
+#include <fsl_vref.h>
```

Comment:
is this needed? I don't see anything from fsl_vref.h being used in this file

Reply:
removed